test/parallel/test-debugger-pid.js

```diff
@@ -29,7 +29,7 @@ interfacer.on('line', function(line) {
     switch (++lineCount) {
       case 1:
         line = line.replace(/^(debug> *)+/, '');
-        var msg = 'There was an internal error in Node\'s debugger. ' +
+        const msg = 'There was an internal error in Node\'s debugger. ' +
                   'Please report this bug.';
```

Comment:
@larissayvette hm, do the `'` line up vertically? If so, I can’t quite tell what the problem is… :/